### ISSUE TITLE
Chat message validation failure

### ISSUE THREAD

**Issue by JohanVonElectrum** [NEZNAMY/TAB#833]
### Server version

BungeeCord #1679

### TAB version

3.3.0

### Describe the bug

I have installed TAB 3.3.0 in a [BungeeCord](https://ci.md-5.net/job/BungeeCord/1679/artifact/bootstrap/target/BungeeCord.jar) server with the [latest](https://ci.md-5.net/job/BungeeCord/1679/artifact/bootstrap/target/BungeeCord.jar) version.
The backend server is running [Paper 1.19.3 422](https://api.papermc.io/v2/projects/paper/versions/1.19.3/builds/422/downloads/paper-1.19.3-422.jar) with no plugins.
I can join, chat and play in general with TAB working, but after `/btab reload` if you chat you will be kicked out with a `Chat message validation failure` error. Not always after `/btab reload` and chat you get kicked out, but almost always (no more than 2 or 3 attempts).

### Expected behavior

Not kicking me, I suppouse.

### Steps to reproduce

1. Start servers (BungeeCord and Paper).
2. `/btab reload`
3. Chat
4. If you are not kicked, go to step 2.
5. After no more than 3 attempts, you are kicked out.

### Additional info

The config is default, only added an animation and changed lines in header and footer. All works normal with the config.

### Checklist

- [X] I am running latest version of the plugin
- [X] I have read the wiki to make sure it's not an issue with configuration
- [ ] I ticked all of the boxes without actually reading them
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform

**Comment by NEZNAMY** [comment]
I enabled online mode on my bungeecord and couldn't even connect without any plugins on neither bungeecord nor paper.
```
18:31:15 [INFO] Using standard Java JCE cipher.
18:31:15 [INFO] Using standard Java compressor.
18:31:15 [INFO] Enabled BungeeCord version git:BungeeCord-Bootstrap:1.19-R0.1-SNAPSHOT:1b6d845:1679
18:31:15 [INFO] Discovered module: ModuleSpec(name=cmd_server, file=modules\cmd_server.jar, provider=JenkinsModuleSource())

18:31:15 [INFO] Loaded plugin cmd_server version git:cmd_server:1.19-R0.1-SNAPSHOT:1b6d845:1679 by SpigotMC
18:31:15 [INFO] Enabled plugin cmd_server version git:cmd_server:1.19-R0.1-SNAPSHOT:1b6d845:1679 by SpigotMC
18:31:16 [INFO] Listening on /0.0.0.0:25577
18:31:50 [INFO] [_NEZNAMY_,/127.0.0.1:52458] <-> InitialHandler has connected
18:31:50 [INFO] [/127.0.0.1:52456] <-> InitialHandler has pinged
18:31:51 [INFO] [_NEZNAMY_] <-> ServerConnector [lobby] has connected
18:31:52 [INFO] Netty is not using direct IO buffers.

18:31:54 [INFO] [_NEZNAMY_] disconnected with: §fmultiplayer.disconnect.invalid_public_key_signature
18:31:54 [INFO] [_NEZNAMY_] -> UpstreamBridge has disconnected
18:31:54 [INFO] [_NEZNAMY_] <-> DownstreamBridge <-> [lobby] has disconnected
```
![image](https://user-images.githubusercontent.com/6338394/221426630-515f088b-3a31-4e98-91ea-df09e26ba48a.png)

Have you tried without TAB? Do you have any idea why am I unable to connect with online mode enabled?

**Comment by JohanVonElectrum** [comment]
I'm using IP forwarding and all of that stuff between bungee and paper. Normal configuration.
Try using `enforce-secure-profile=false` in server.properties.
Without TAB I get the blue popup on join, but can't replicate the bug that kicks me out.

**Comment by NEZNAMY** [comment]
Same issue.

**Comment by JohanVonElectrum** [comment]
I fixed my issue using the plugin [AntiPopup](https://github.com/KaspianDev/AntiPopup/releases/tag/4444b79).

**Comment by NEZNAMY** [comment]
Do you want to investigate this further?

**Comment by NEZNAMY** [comment]
Reload command does not send profile key, unless global playerlist is enabled. You say you have default config, where global playerlist is disabled. Have you tried without TAB? What if you don't use the reload command?

**Comment by NEZNAMY** [comment]
@JohanVonElectrum are you still interested in investigating this issue?

**Comment by JohanVonElectrum** [comment]
It would be nice for other people who can't use the plugin I used to fix it, but I'm not interested in this issue anymore.

**Comment by NEZNAMY** [comment]
I was finally able to connect to an online mode bungeecord. After trying out literally everything, I can confirm that this is **not** a TAB bug. It happens after executing a command from **any** bungeecord plugin. Since I had to give myself permissions for /btab reload, I had luckperms installed, so I tried out that one. Same result using the following steps:
* Joined online mode BungeeCord with only 1 plugin, backend 0 plugins
* `/lpb`
* \<chat\>
* `/lpb`
* \<chat\>
* Got kicked with console message `[Netty Server IO #1/WARN]: Failed to validate message acknowledgements from _NEZNAMY_`

This proves that this is not a TAB bug, but probably a BungeeCord bug. I heard since 1.19.1 commands cannot be cancelled anymore or something like that, could be related. Unfortunately I am not able to help you with this issue. Try opening an issue on BungeeCord issue tracker. They might be of more help.
